### ISSUE TITLE
Docker: Alpine build is failing

### ISSUE THREAD

**Issue by Nicceboy** [radareorg/radare2#15781]
# Docker build for Alpine base is failing
There is already similar issue in Cutter: https://github.com/radareorg/cutter/issues/1908
However, I though it would be relevant to add here as well, since it seems to be Radare2's problem.

### Work environment

| Questions                                            | Answers
|------------------------------------------------------|--------------------
| OS/arch/bits (mandatory)                             | Arch Linux x86_64 GNU/Linux 5.4.7-arch1-1
|Docker version |19.03.5-ce, build 633a0ea838

git.2.24.1 commit: 0de7c21bcc53515f0c339bd50e14ca8c06cdd99e
Happens with the release of 4.1.1 as well

### Expected behavior
Docker image should be built.
### Actual behavior
Build is failing:
```
p/debug_native.c:95:12: warning: 'r_debug_handle_signals' defined but not used [-Wunused-function]
   95 | static int r_debug_handle_signals(RDebug *dbg) {
      |            ^~~~~~~~~~~~~~~~~~~~~~
CC linux_coredump.c
p/native/linux/linux_debug.c: In function 'linux_reg_profile':
p/native/linux/linux_debug.c:43:10: fatal error: bits/sigcontext.h: No such file or directory
   43 | #include <bits/sigcontext.h>
      |          ^~~~~~~~~~~~~~~~~~~
compilation terminated.
make[4]: *** [/mnt/radare2/libr/../global.mk:37: p/native/linux/linux_debug.o] Error 1
make[4]: *** Waiting for unfinished jobs....
p/native/linux/linux_coredump.c: In function 'linux_get_prstatus':
p/native/linux/linux_coredump.c:228:54: warning: passing argument 5 of 'r_debug_ptrace' makes integer from pointer without a cast [-Wint-conversion]
  228 |  if (r_debug_ptrace (dbg, PTRACE_GETREGS, tid, NULL, &regs) < 0) {
      |                                                      ^~~~~
      |                                                      |
      |                                                      long long unsigned int (*)[27]
In file included from p/native/linux/linux_coredump.c:3:
/mnt/radare2/libr/include/r_debug.h:614:115: note: expected 'r_ptrace_data_t' {aka 'int'} but argument is of type 'long long unsigned int (*)[27]'
  614 | static inline long r_debug_ptrace(RDebug *dbg, r_ptrace_request_t request, pid_t pid, void *addr, r_ptrace_data_t data) {
      |                                                                                                   ~~~~~~~~~~~~~~~~^~~~
p/native/linux/linux_coredump.c: In function 'linux_get_fp_regset':
p/native/linux/linux_coredump.c:242:57: warning: passing argument 5 of 'r_debug_ptrace' makes integer from pointer without a cast [-Wint-conversion]
  242 |   if (r_debug_ptrace (dbg, PTRACE_GETFPREGS, pid, NULL, p) < 0) {
      |                                                         ^
      |                                                         |
      |                                                         elf_fpregset_t * {aka struct user_fpregs_struct *}
In file included from p/native/linux/linux_coredump.c:3:
/mnt/radare2/libr/include/r_debug.h:614:115: note: expected 'r_ptrace_data_t' {aka 'int'} but argument is of type 'elf_fpregset_t *' {aka 'struct user_fpregs_struct *'}
  614 | static inline long r_debug_ptrace(RDebug *dbg, r_ptrace_request_t request, pid_t pid, void *addr, r_ptrace_data_t data) {
      |                                                                                                   ~~~~~~~~~~~~~~~~^~~~
p/native/linux/linux_coredump.c: In function 'linux_get_xsave_data':
p/native/linux/linux_coredump.c:938:73: warning: passing argument 5 of 'r_debug_ptrace' makes integer from pointer without a cast [-Wint-conversion]
  938 |  if (r_debug_ptrace (dbg, PTRACE_GETREGSET, tid, (void *)NT_X86_XSTATE, &transfer) < 0) {
      |                                                                         ^~~~~~~~~
      |                                                                         |
      |                                                                         struct iovec *
In file included from p/native/linux/linux_coredump.c:3:
/mnt/radare2/libr/include/r_debug.h:614:115: note: expected 'r_ptrace_data_t' {aka 'int'} but argument is of type 'struct iovec *'
  614 | static inline long r_debug_ptrace(RDebug *dbg, r_ptrace_request_t request, pid_t pid, void *addr, r_ptrace_data_t data) {
      |                                                                                                   ~~~~~~~~~~~~~~~~^~~~
p/native/linux/linux_coredump.c: In function 'get_xsave_size':
p/native/linux/linux_coredump.c:1348:73: warning: passing argument 5 of 'r_debug_ptrace' makes integer from pointer without a cast [-Wint-conversion]
 1348 |  if (r_debug_ptrace (dbg, PTRACE_GETREGSET, pid, (void *)NT_X86_XSTATE, &local) < 0) {
      |                                                                         ^~~~~~
      |                                                                         |
      |                                                                         struct iovec *
In file included from p/native/linux/linux_coredump.c:3:
/mnt/radare2/libr/include/r_debug.h:614:115: note: expected 'r_ptrace_data_t' {aka 'int'} but argument is of type 'struct iovec *'
  614 | static inline long r_debug_ptrace(RDebug *dbg, r_ptrace_request_t request, pid_t pid, void *addr, r_ptrace_data_t data) {
      |                                                                                                   ~~~~~~~~~~~~~~~~^~~~
make[3]: *** [Makefile:13: foo] Error 2
make[2]: *** [Makefile:152: debug] Error 2
make[1]: *** [Makefile:28: all] Error 2
make: *** [Makefile:60: all] Error 2
```

### Steps to reproduce the behavior 
Run Alpine build script: https://github.com/radareorg/radare2/blob/master/sys/docker_build_alpine_image.sh

**Comment by radare** [comment]
interesting. thanks! something to fix for 4.2

**Comment by radare** [comment]
fixed in master 0e34303648430dfcb3f0b9947233d897883e1484
